examples/falcon_sensor_download/main.go

```diff
@@ -140,7 +140,15 @@ func getSensors(client *client.CrowdStrikeAPISpecification, osName string) []*mo
 		panic("Errors from the server")
 
 	}
-	return payload.Resources
+	k := 0
+	for _, sensor := range payload.Resources {
+		if strings.Contains(*sensor.Description, "Falcon SIEM Connector") {
```

Comment:
@isimluk I wonder if there are other downloads available... like the GPG key?

Reply:
Looking at the output... there isn't... weird.

Reply:
Not under sensor download api.